src/platform-apps/channels/channel-view.test.tsx

```diff
@@ -40,10 +40,46 @@ describe('ChannelView', () => {
     ]);
   });
 
+  it('renders header date', () => {
+    const messages = [
+      { id: 'message-one', message: 'what', createdAt: 1658776625730 },
+      { id: 'message-two', message: 'hello', createdAt: 1658776625730 },
+    ];
+
+    const wrapper = subject({ messages });
+
+    expect(wrapper.find('.message__header-date').exists()).toBe(true);
+  });
+
+  it('renders a header Date grouped by day', () => {
+    const messages = [
+      { id: 'message-one', message: 'what', createdAt: 1658776625730 },
+      { id: 'message-two', message: 'hello', createdAt: 1658776625730 },
+      { id: 'message-three', message: 'who', createdAt: 1659018545428 },
+      { id: 'message-for', message: 'there', createdAt: 1659655757948 },
+    ];
+
+    const wrapper = subject({ messages });
+
+    expect(wrapper.find('.message__header-date').length).toStrictEqual(3);
+  });
+
+  it('renders a header Date contain Today', () => {
+    const messages = [
+      { id: 'message-one', message: 'what', createdAt: new Date() },
```

Comment:
what is the type of createdAt from the server? let's make sure that we know what it is, and that we keep it consistent. above we're using timestamps, and here we're using a date object. let's pick one (either what the server returns, or convert it to something specific when we build the initial object) and use it consistently.